build.sh

```diff
@@ -42,17 +44,19 @@ fi
 # INSTALL .NET CORE CLI
 ###########################################################################
 
-echo "Installing .NET CLI..."
```

Comment:
This change for not installing .NET CLI is whats causing the issue here. Revert this section and it should solve your problem